pkg/network/tracer/tracer_classification_test.go

```diff
@@ -1314,7 +1313,7 @@ func testAMQPProtocolClassification(t *testing.T, cfg *config.Config, clientHost
 				require.NoError(t, client.DeclareQueue("test", client.PublishChannel))
 			},
 			teardown:   amqpTeardown,
-			validation: validateProtocolConnection(network.ProtocolUnknown),
+			validation: validateProtocolConnection(network.ProtocolAMQP),
```

Comment:
I changed this because it seemed incorrect before, but please let me know if that isn't true.

Reply:
we shouldn't classify that message type